src/mpid/ch4/shm/ipc/src/ipc_p2p.h

```diff
@@ -51,17 +50,14 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_IPC_send_contig_lmt(const void *buf, MPI_Aint
     MPIDIG_REQUEST(sreq, count) = count;
```

Comment:
We might want to update the function description comment too, to match the mem_attr change.

Reply:
right